end-to-end-test/remote/specs/core/groupComparisonLollipop.spec.js

```diff
@@ -0,0 +1,234 @@
+var assert = require('assert');
+var goToUrlAndSetLocalStorage = require('../../../shared/specUtils')
+    .goToUrlAndSetLocalStorage;
+var waitForNetworkQuiet = require('../../../shared/specUtils')
+    .waitForNetworkQuiet;
+var setInputText = require('../../../shared/specUtils').setInputText;
+var setSettingsMenuOpen = require('../../../shared/specUtils')
+    .setSettingsMenuOpen;
+const { getElementByTestHandle } = require('../../../shared/specUtils');
+
+const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, '');
+
+describe('group comparison mutations tab tests', function() {
+    describe('lollipop alerts and plot display', function() {
+        before(function() {
+            goToUrlAndSetLocalStorage(
+                `${CBIOPORTAL_URL}/comparison?comparisonId=634006c24dd45f2bc4c3d4aa`
+            );
+            $('div[data-test="ComparisonPageOverlapTabDiv"]').waitForExist();
+        });
+
+        it('too many groups alert displayed when more than 2 groups selected', function() {
+            $('a.tabAnchor_mutations').waitForExist();
+            $('a.tabAnchor_mutations').click();
+            browser.pause(1000);
```

Comment:
we should avoid using pauses in order to minimize test duration.   instead, just do ```$(..).waitForDisplayed()```
Note that you **do not actually have to assert anything after this**.  If waitForDisplayed times out, the test will fail.   If it succeeds, the test will pass.